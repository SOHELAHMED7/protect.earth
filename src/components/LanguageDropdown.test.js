import React from "react";
import renderer from "react-test-renderer";

import LanguageDropdown from "./LanguageDropdown";

describe("LanguageDropdown", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<LanguageDropdown />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
