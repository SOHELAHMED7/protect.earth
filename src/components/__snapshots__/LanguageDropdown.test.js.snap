// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguageDropdown renders correctly 1`] = `
<div
  className="change-language dropdown"
  onKeyDown={[Function]}
>
  <button
    aria-expanded={false}
    aria-haspopup={true}
    className="dropdown-toggle btn btn-primary"
    disabled={false}
    id="dropdown-language"
    onClick={[Function]}
    type="button"
  >
    Select language
  </button>
</div>
`;
